db/import-tmpl/produce_db.yml

```diff
@@ -0,0 +1,49 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2021 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Item Produce Database
+###########################################################################
+#
+# Item Produce Settings
+#
+###########################################################################
+# - ItemLevel                  Number which determines what kind of a crafting window will pop-up.
+#   Recipe:
+#    - Product                 AegisName of the produced item.
+#      Skill:                  Skill requirement. (Default: null)
+#        Name                  Skill name required.
+#        Level                 Skill level required.
+#      Consumed:               List of items consumed to produce the Product.
+#        - Item                AegisName of the consumed item.
+#          Amount              Amount required.
+#          Clear               Remove the item with the given AegisName from Consumed. (Optional)
+#      NotConsumed:            List of items not consumed to produce the Product. (Default: null)
+#        - Item                AegisName of the unconsumed item.
+#          Clear               Remove the item with the given AegisName from NotConsumed. (Optional)
```

Comment:
Why dont we combine these two and make "Consumed" a flag (Default: true?) ?

Reply:
I found it clearer to separate the 2 because Amount is required for Consumed and is not required for NotConsumed than add an exception in the description

Reply:
Another option would be:
```yaml
  Recipe:
    Catalysts:
      - Item
        Amount
        Consumed    Whether the item is consumed or not. (Default: true)
        Clear
```